scss/_variables.scss

```diff
@@ -5,6 +5,7 @@ $heading-color: #333333;
 $text-color: #666666;
 
 /// Fonts Variables
-$accent-font: "Raleway", sans-serif;
-$heading-font: "Raleway", sans-serif;
-$text-font: "Open Sans", sans-serif;
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700');
```

Comment:
can you check to make sure that Open Sans is not imported elsewhere?

Also remove the 400 and 600. We only use 300 and 700 on this site from what I have seen.

Please double check before removing 🙈